### DISCUSSION TITLE
构建服务器集群，夸分支的时候，选择到其他构建服务器了，能否一个流水线中，构建服务器选择的是一台服务器

### DISCUSSION THREAD

**Discussion by pgy0112** [Tencent/bk-ci#5122] [General]
构建服务器集群，夸分支的时候，选择到其他构建服务器了，能否一个流水线中，构建服务器选择的是一台服务器

**Comment by pgy0112** [comment]
比如有三台构建服务器，流水线中如果前面选择的是其中1台，后面的分支，是否也可以选择前面这台服务器呢？ 要不编译后的文件，后面镜像发布的时候，如果选择到了其他构建服务器，会找不到对应的文件

**Comment by pgy0112** [comment]
内部提单，可以联系kevingypeng

**Comment by iccengan** [comment]
通过 #1122 解决
